### ISSUE TITLE
AutoRefresh menu not accessible via keyboard

### ISSUE THREAD

**Issue by JoelHaslam** [grafana/grafana#39838]
As mentioned on a different thread, I'm aware that there are various accessibility issues currently being worked on. I can't find an issue for this and wanted to bring attention to it, but sorry if I've missed something!

**Steps to reproduce**:
Press tab until keyboard focus reaches the _down arrow_ icon located next to the 'refresh dashboard' icon. Using this, try to change the auto refresh time / turn auto refresh off using the keyboard.

**Actual Result**:
https://user-images.githubusercontent.com/63413067/135409097-8607e56d-3c5e-4f6b-b6df-9ab5c1f0cb8c.mp4

As you can see from the video this is very difficult (/impossible) to do. At first I was pressing enter causing the menu to only appear for a very brief moment. Next I pressed space to get the menu to open on a more permanent basis, however I was unable to get keyboard focus into the menu to select a new option.

**Expected Result**:
I should be able to at least turn off auto refresh using only the keyboard, this can be a very useful tool when using Grafana.

**Relevant WCAG Criteria:** [2.2.1 timing-adjustable](https://www.w3.org/TR/WCAG21/#timing-adjustable) / [2.2.2 pause-stop-hide](https://www.w3.org/TR/WCAG21/#pause-stop-hide)

**Environment**:
- Grafana version: 8.2.0-beta1
- Data source type & version: Default data used at [play.grafana.org](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1)
- User OS & Browser: Windows & Chrome
- Others:

**Comment by hugohaggmark** [comment]
Related to #35613

**Comment by JoelHaslam** [comment]
Hi @anavss, I know it's been a while, but are you still planning to hold any more usability chats with the community? I'd love get involved if so!

**Comment by anavss** [comment]
Hi @JoelHaslam, absolutely! Your message came right on time as we are resuming the navigation research this week. I will be in touch via email to book a chat. 🙂
